### PR TITLE
added info on using curl

### PR DESCRIPTION
**Pull request by CodeWriterWriter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,12 @@ This micro-service recognizes the following messages:
   * _role:github,cmd:query_
   * _role:github,cmd:parse_
 
+## Running with Curl
 
+Any of the messages above can be run using curl in the following format
+```
+curl -d '{"role":"github","cmd":"get"}' http://localhost:52472/act
```
Comment (mcdonnelldean) [inline README.md]: Super stuff, we need to add instructions on how to find the port number as it will change from run to to run

Comment (CodeWriterWriter) [inline README.md]: I found it by editing the mesh.js to print the port number, but I don't think that would be reasonable to expect an end user to do

Comment (mcdonnelldean) [inline README.md]: Can you show me the line? I'll figure out how to get it exposed

Comment (CodeWriterWriter) [inline README.md]: function join( instance, config, done ) {
    config = config || {}

    if( !config.pin && !config.pins ) {
      config.pin = 'null:true'
    }
    console.log(config.port)

**Comment by mcdonnelldean** [comment]
@CodeWriterWriter I'm going to merge for now, until we figure out how to log the port it's fine

**Pull request merged**